src/main/steps/common/common.content.ts

```diff
@@ -158,7 +158,8 @@ export const generatePageContent = (
   language: Language,
   pageContent?: TranslationFn,
   isDivorce = true,
-  formState?: Partial<Case>
+  formState?: Partial<Case>,
+  userEmail?: string
```

Comment:
as above, I'm wondering when it's not set?

Reply:
👍 Good spot, I've made this required in e21275c.